spotbugs/src/main/java/edu/umd/cs/findbugs/SAXBugCollectionHandler.java

```diff
@@ -65,6 +65,9 @@
  * @author David Hovemeyer
  */
 public class SAXBugCollectionHandler extends DefaultHandler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PluginLoader.class);
```

Comment:
This seems like the wrong logger class. Perhaps consider using `MethodHandles.lookup().lookupClass()` to dynamically detect the enclosing class.

Reply:
Yes, I corrected it.